tests/orion/api/test_deployments.py

```diff
@@ -101,6 +101,15 @@ async def test_create_deployment(
         assert deployment.infrastructure_document_id == infrastructure_document_id
         assert deployment.storage_document_id == storage_document_id
 
+    async def test_default_work_queue_name_is_none(self, session, client, flow):
+
+        data = DeploymentCreate(
+            name="My Deployment", manifest_path="", flow_id=flow.id
+        ).dict(json_compatible=True)
+        response = await client.post("/deployments/", json=data)
+        assert response.status_code == status.HTTP_201_CREATED
+        assert response.json()["work_queue_name"] == None
```

Comment:
Hmmm I thought the default value for work queue name was `"global"`

Reply:
See discussion [here](https://github.com/PrefectHQ/prefect/pull/6317#issuecomment-1209911886) -- I need to update the description of the PR!